tests/integration/common.rs

```diff
@@ -230,8 +239,22 @@ pub async fn wait_for_tx(client: &mut TestClient, transaction_id: TransactionId)
             break;
         }
 
-        // 500_000_000 ns = 0.5s
-        std::thread::sleep(std::time::Duration::new(0, 500_000_000));
+        // 100_000_000 ns = 0.1s
+        std::thread::sleep(std::time::Duration::new(0, 100_000_000));
+    }
+    if std::env::var("LOG_WAIT_TIMES").unwrap_or_else(|_| "false".to_string()) == "true" {
```

Comment:
nit: This could probably 
```suggestion
    if let Ok("true".into()) = std::env::var("LOG_WAIT_TIMES") {
```

Reply:
Also, can you add some comments here to explain why this is here and how it works?